flex-checks/src/main/java/org/sonar/flex/checks/ConditionalStructure.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * SonarQube Flex Plugin
+ * Copyright (C) 2010-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.flex.checks;
+
+import com.sonar.sslr.api.AstNode;
+import com.sonar.sslr.api.Token;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import org.sonar.flex.FlexGrammar;
+import org.sonar.flex.FlexKeyword;
+
+import static java.util.Collections.singletonList;
+
+class ConditionalStructure {
+  static class BranchAndContent {
+    AstNode branch;
+    List<AstNode> content;
+    boolean onLiner;
```

Comment:
Please add an "e": oneLiner.

Reply:
oops. good eyes you have. Bad fingers I do.

Reply:
fixed